roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -30,12 +30,18 @@ cpu_allocation_ratio={{ nova.cpu_allocation_ratio }}
 ram_allocation_ratio={{ nova.ram_allocation_ratio }}
 {% endif -%}
 
+#Reserve CPUs on H8DGU or 64 core hosts
 {% if ansible_product_name == 'H8DGU' and ansible_processor_vcpus == 32 and inventory_hostname in groups['controller'] -%}
 vcpu_pin_set = "2-15,18-31"
 {% elif ansible_product_name == 'H8DGU' and ansible_processor_vcpus == 32 -%}
 vcpu_pin_set = "1-31"
+{% elif ansible_processor_vcpus == 64 and inventory_hostname in groups['controller'] -%}
+vcpu_pin_set = "2-31,33-63"
+{% elif ansible_processor_vcpus == 64 -%}
+vcpu_pin_set = "1-63"
 {% endif -%}
```

Comment:
So existing code reserves two cores (one [https://en.wikipedia.org/wiki/Bulldozer_(microarchitecture)#CMT CMT unit]) on each socket for this HW platform. If we are going to reserve cores on a wide range of HW platforms, we should not try such optimizations and use a formula to reserve 1 CPU core on non-controller nodes, and 4 CPU cores on controller nodes.